src/api/json/catalog.json

```diff
@@ -16,13 +16,13 @@
     {
       "name": "Application Accelerator",
       "description": "Application Accelerator for VMware Tanzu",
-      "fileMatch": ["accelerator.yaml"],
```

Comment:
Looks like you've formatted the entire file. Please follow the formatting that was in place, and only add the entry required.